continuous_integration/install.sh

```diff
@@ -76,3 +76,4 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 python setup.py install
+pip install nose-exclude
```

Comment:
> Ah I couldn't see any tests in dill but I missed the fact that they were
> doctests.

You need to patch the doctests. Sorry. You can look at 
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/externals/copy_joblib.sh
The search and replace is done in the first "find".